include/gz/math/Material.hh

```diff
@@ -40,7 +40,7 @@ namespace gz
     /// enum.
     ///
     /// This class will replace the
-    /// [MaterialDensity class](https://github.com/ignitionrobotics/ign-common/blob/ign-common1/include/gz/common/MaterialDensity.hh)
+    /// [MaterialDensity class](https://github.com/gazebosim/gz-common/blob/ign-common1/include/gz/common/MaterialDensity.hh)
```

Comment:
I just noticed this link was previously broken because the `include/gz` folder doesn't exist on the `ign-common1` branch

Reply:
Should I point it to main or point `gz` back to `ignition`?

Reply:
It can point to `main`

https://github.com/gazebosim/gz-common/blob/main/include/gz/common/MaterialDensity.hh